pkg/server/updateResults.go

```diff
@@ -128,7 +133,7 @@ func postUpdateResult(updateResults []UpdateResults) error {
 func getTimeWindows(days int, windowSize int) []time.Time {
 	var windows []time.Time
 	currentTime := time.Now()                  // To Do : we should get the time from the input
-	startTime := currentTime.AddDate(0, 0, -1) // Goback to one day
+	startTime := currentTime.AddDate(0, 0, -5) // Goback to one day
```

Comment:
@Anxhela21 Please update the comment for Going back to 5 days . This has to be a day from the input

Reply:
updated comment. Will address the input TODO in another PR.